tests/proposals_tests.cairo

```diff
@@ -162,3 +165,128 @@ fn test_submit_proposal_under_quorum() {
     );
     dispatcher.submit_proposal(42, 1);
 }
+
+#[test]
+#[should_panic(expected: ('Proposal is not live!',))]
+fn test_add_comment_on_non_live_proposal() {
+    let token_contract = deploy_and_distribute_gov_tokens(admin_addr.try_into().unwrap());
+    let gov_contract = deploy_governance(token_contract.contract_address);
+    let gov_contract_addr = gov_contract.contract_address;
+    let ipfs_hash = 1234567;
```

Comment:
See, had you tested with a real ipfs hash, you would catch that it won't fit in felt252 ;)